src/app/rpmostree-ex-builtin-commit2jigdo.c

```diff
@@ -611,7 +611,7 @@ generate_spec (RpmOstreeCommit2JigdoContext  *self,
   g_autoptr(GString) replacement = g_string_new ("");
   g_string_append_len (replacement, spec_contents, meta - spec_contents);
   g_string_append (replacement, "# Generated by rpm-ostree\n");
-  g_string_append (replacement, "Provides: " RPMOSTREE_JIGDO_PROVIDE_V2 "\n");
+  g_string_append (replacement, "Provides: " RPMOSTREE_JIGDO_PROVIDE_V3 "\n");
```

Comment:
Minor: since this is the same PR, there's no harm in leaving this at `V2`, right?

Reply:
Yeah, I was trying to preserve the history.  I could squash them together?  Dunno.  No harm in incrementing the number right?